geotrek/core/tests/test_commands.py

```diff
@@ -229,3 +230,353 @@ def test_load_paths_within_spatial_extent_no_srid_geom(self):
         value = Path.objects.first()
         self.assertEqual(value.name, 'lulu')
         self.assertEqual(value.structure, self.structure)
+
+
+@skipIf(not settings.TREKKING_TOPOLOGY_ENABLED, 'Test with dynamic segmentation only')
+class ReorderTopologiesPathAggregationTest(TestCase):
+    def setUp(self):
+        """
+        ⠳               ⠞
+          ⠳           ⠞
+            ⠳       ⠞
+              ⠳   ⠞
+                ⠿
+              ⠞   ⠳
+            ⠞       ⠳
+        1 ⠞           ⠳ 2
+        ⠞               ⠳
+        """
+        self.path_1 = PathFactory.create(geom=LineString(Point(700000, 6600000), Point(700100, 6600100),
+                                                         srid=settings.SRID))
+        self.path_2 = PathFactory.create(geom=LineString(Point(700000, 6600100), Point(700100, 6600000),
+                                                         srid=settings.SRID))
+        self.path_1_a = Path.objects.get(geom=LineString(Point(700000, 6600000), Point(700050, 6600050),
+                                                         srid=settings.SRID))
+        self.path_1_b = Path.objects.get(geom=LineString(Point(700050, 6600050), Point(700100, 6600100),
+                                                         srid=settings.SRID))
+        self.path_2_a = Path.objects.get(geom=LineString(Point(700000, 6600100), Point(700050, 6600050),
+                                                         srid=settings.SRID))
+        self.path_2_b = Path.objects.get(geom=LineString(Point(700050, 6600050), Point(700100, 6600000),
+                                                         srid=settings.SRID))
+
+    def get_geometries(self):
+        geometries = []
+        for pathagg in PathAggregation.objects.all():
+            cursor = connection.cursor()
+            cursor.execute(f"""SELECT * FROM ST_ASTEXT(ST_SmartLineSubstring('{pathagg.path.geom.wkt}'::geometry,
+                                                                              {pathagg.start_position},
+                                                                              {pathagg.end_position}
+                                                       ))
+            """)
+            geom = cursor.fetchall()[0][0]
+            geometries.append(GEOSGeometry(geom, srid=2154))
+        return geometries
+
+    def test_split_reorder_1(self):
+        """
+        Part A
+
+        ⠳               🡥
```

Comment:
No big deal because it has something to do with my setup : the up arrow character is not displayed neither on Github or PyCharm for me. What did you use?